examples/plot_baby_afq.py

```diff
@@ -0,0 +1,496 @@
+                             pediatric_templates[name + '_roi2' + hemi]],
+                    'rules': [True, True],
+                    'cross_midline': False,
+                    # reuse probability map from ILF
+                    'prob_map': prob_map_data[...,
+                                              prob_map_order.index("ILF" + hemi)],
+                    'uid': uid}
+                uid += 1
+        # Default: two ROIs within hemisphere
+        else:
+            for hemi in ['_R', '_L']:
+                pediatric_bundles[name + hemi] = {
+                    'ROIs': [pediatric_templates[name + '_roi1' + hemi],
+                             pediatric_templates[name + '_roi2' + hemi]],
+                    'rules': [True, True],
+                    'cross_midline': False,
+                    'prob_map': prob_map_data[...,
+                                              prob_map_order.index(name + hemi)],
+                    'uid': uid}
+                uid += 1
+
+    return pediatric_bundles
+
+
+# monkey patch
+api.make_bundle_dict = make_pediatric_bundle_dict
+
+##########################################################################
+# Get example data:
+# -----------------
+# **Diffusion dataset**
+#
+# .. note::
+#   The diffusion data used in this example are from the Developing Human
+#   Connectome project (dHCP):
+#
+#   http://www.developingconnectome.org/project/
+#
+#   Acknowledgement: These results were obtained using data made available from
+#   the Developing Human Connectome Project funded by the European Research
+#   Council under the European Union’s Seventh Framework Programme
+#   (FP/2007-2013) / ERC Grant Agreement no. [319456]
+#
+# .. note::
+#   This example assumes:
+#
+#   - A single subject and session
+#
+#   - Subject's session data has been downloaded into
+#
+#     `AFQ_data\\dHCP\\derivatives`
+#
+#   - Subject's session data has been made BIDS compliant
+#
+#     see https://bids.neuroimaging.io
+#
+
+print("Validating data...")
+
+dhcp_home = op.join(afq_home, 'dHCP')
+
+myafq = api.AFQ(# ==== BIDS parameters ====
+                bids_path=dhcp_home,
+                dmriprep='dHCP neonatal dMRI pipeline',
+                # bids_filters={"suffix": "dwi"}, # default
+                # custom_tractography_bids_filters=None, # default
+                # ===== Registration parameters ====
+                # b0_threshold=50, # default
+                # min_bval=None, # default
+                min_bval=1000,  # override
+                # max_bval=None, # default
+                max_bval=1000,  # override
+                reg_template=pediatric_templates['UNCNeo-withCerebellum-for-babyAFQ'],
+                # reg_subject="power_map", # default
+                reg_subject="b0",  # override
+                # brain_mask=B0Mask(), # default
+                brain_mask=MaskFile("brainmask",
+                                    {"scope": "dHCP neonatal dMRI pipeline"}),
```

Comment:
Yep - this is it!

Check if this is a binary mask or a continuous probability mask, and then threshold if needed (there is a class for that!).

Reply:
Just so I understand what is going on: Which registration algorithm are we using? Are we applying a brain mask in the step above or are we just checking that it is not a probability mask?

Reply:
`reg_algo=None` by default, which should use Symmetric Diffeomorphic Registration (SyN) Algorithm for registration.

In this step we are configuring pyAFQ and nothing is actually run until the `myafq.export_all()` command. According to the docs the `brain_mask` is applied before registration to a template.